credentials/src/main/kotlin/SSI.kt

```diff
@@ -137,6 +137,7 @@ public data class CreateVcOptions(
 public data class CreateVpOptions(
   val presentationDefinition: PresentationDefinitionV2,
   val verifiableCredentialJwts: List<VcJwt>,
+  val resolver: DIDResolver,
```

Comment:
It's strange to me to accept a `resolver` as part of a Creation option. That said, can you document what each of the options in the `CreateVpOptions` does?

Reply:
Yes I think the resolver is strange too, the only reason it exists here is because we do a 2nd verification. of the VC.

I was thinking about mkaing it optional, and if not set then dont verify the VCJwts..

That could be strange though because we could potentially make a VP with invalid VCs..

Reply:
A holder is typically the one who creates a VP. As such, they probably don't care much about the verification of credentials that are contained within it. The Verifier is the one who is going to be verifying both the VP and all the VCs contained inside it.

Given the above, what do you think about removing the option of verifying the VCs when creating the VP? If the holder DOES want to do so, they can always call the verify methods separately.

Reply:
Yup I agree, I'll change this to not have the resolver and leave the verifying for the verifyer